va/caa_test.go

```diff
@@ -209,193 +209,193 @@ func TestCAAChecking(t *testing.T) {
 	testCases := []struct {
 		Name    string
 		Domain  string
-		Present bool
+		FoundAt string
 		Valid   bool
 	}{
 		{
 			Name:    "Bad (Reserved)",
 			Domain:  "reserved.com",
-			Present: true,
+			FoundAt: "reserved.com",
 			Valid:   false,
 		},
 		{
 			Name:    "Bad (Reserved, Mixed case Issue)",
 			Domain:  "mixedcase.com",
-			Present: true,
+			FoundAt: "mixedcase.com",
 			Valid:   false,
 		},
 		{
 			Name:    "Bad (Critical)",
 			Domain:  "critical.com",
-			Present: true,
+			FoundAt: "critical.com",
 			Valid:   false,
 		},
 		{
 			Name:    "Bad (NX Critical)",
 			Domain:  "nx.critical.com",
-			Present: true,
+			FoundAt: "critical.com",
```

Comment:
It looks like this is the only test case that covers `FoundAt` not equal to the FQDN. Want to add one where the answer is good, and FoundAt is a parent domain?

Reply:
Good idea, added!